### ISSUE TITLE
The dynamic text changes are announced multiple times or not announced at all in the Button with JAWS

### ISSUE THREAD

**Issue by havvamehfooz** [FreedomScientific/VFO-standards-support#805]
Summary

The dynamic text changes are announced multiple times in the button with JAWS

The reason behind that is a workaround we’ve done for JAWS with which we overcompensate the missing announcements leading to redundant announcements.

The real issue is that the dynamic text changes (label changes) for a button aren’t always announced by JWAS as observed via the following snippet and native HTML:

The same behavior isn’t reproducible with NVDA or VoiceOver on mac. The dynamic text changes are always announced there.

Example:

•	https://jsfiddle.net/6f2pqhLr/

Workflow:

Steps to reproduce:
1)	Start JAWS
2)	Press the button -> text change gets announced
3)	Press the button again -> text change doesn’t get announced

**Comment by JAWS-test** [comment]
It should be added:
4. Press the button -> text change gets announced
5. Press the button again -> text change doesn’t get announced
6. Press the button -> text change gets announced
7. Press the button again -> text change doesn’t get announced

This means that the original value is not output again, but the new value, also when the display is repeated.
